src/java/edu/slu/tpen/servlet/ProjectServlet.java

```diff
@@ -119,8 +119,19 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
                                 }
                                 resp.setHeader("Access-Control-Allow-Headers", "*");
                                 resp.setHeader("Access-Control-Expose-Headers", "*"); //Headers are restricted, unless you explicitly expose them.  Darn Browsers.
-                                resp.setHeader("Cache-Control", "max-age=15, must-revalidate"); 
-                                resp.getWriter().write(new JsonLDExporter(proj, new User(uid)).export());
+                                resp.setHeader("Cache-Control", "max-age=15, must-revalidate");
+
+                                if (req.getHeader("Accept").contains("iiif/v3"))
```

Comment:
From the Java8 HTTPServletRequest documentation
```
getHeader returns the value of the specified request header as a String. If the request did not include a header of the specified name, this method returns null.
```
null.contains("iiif/v3") will throw a Null Pointer Exception.

If a request comes through that does not contain an Accept header, this will throw a Null Pointer Exception. I think the browser always adds `Accept: */*` or something by default, so this may never actually happen, but you should still account for this possibility and avoid the NPE.